packages/valory/skills/abstract_round_abci/behaviours.py

```diff
@@ -106,7 +106,9 @@ def _check_matching_round_consistency(
         """Check that matching rounds are: (1) unique across behaviour, and (2) covering."""
         round_to_behaviour: Dict[Type[AbstractRound], List[BehaviourType]] = {
             round_cls: []
-            for round_cls in behaviour_cls.abci_app_cls.get_all_round_classes()
+            for round_cls in behaviour_cls.abci_app_cls.get_all_round_classes(
+                behaviour_cls.is_background_behaviour_set
+            )
```

Comment:
We take into account the termination rounds only when we have the background round set.